action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Publish to npm'
```

Comment:
Do our actions get published to npm? I always thought they just lived on GitHub.

Reply:
nope! they just live on GitHub.